benchmarks/benchmarks-stack/benchmarks-stack/cdk.context.json

```diff
@@ -0,0 +1,13 @@
+{
+  "availability-zones:account=123124136734:region=us-west-2": [
+    "us-west-2a",
```

Comment:
What are the AZ's for? :)

Reply:
hmmm, it's auto-generated by CDK, will remove!